src/main/scala/no/ndla/articleapi/service/UpdateService.scala

```diff
@@ -0,0 +1,63 @@
+/*
+ * Part of NDLA article_api.
+ * Copyright (C) 2017 NDLA
+ *
+ * See LICENSE
+ *
+ */
+
+package no.ndla.articleapi.service
+
+import no.ndla.articleapi.model.api
+import no.ndla.articleapi.model.domain
+import no.ndla.articleapi.model.domain.LanguageField
+import no.ndla.articleapi.repository.ArticleRepository
+import no.ndla.articleapi.validation.ArticleValidator
+
+trait UpdateService {
+  this: ArticleRepository with ConverterService with ArticleValidator with Clock =>
+  val updateService: UpdateService
+
+  class UpdateService {
+    def newArticle(newArticle: api.NewArticle) = {
```

Comment:
Hmm, kan gjerne kalle den `StorageService` men da vil jeg putte `ReadService` (brukes for å lese ting fra database) inn i `StorageService` også.
Eventuelt kan `UpdateService` renames til `WriteService` som står litt mer i stil til `ReadService` som vi har fra før.